src/app/modules/lazy-image/lazy-image.component.ts

```diff
@@ -8,26 +8,23 @@ import {
     OnInit,
     TemplateRef,
     ViewChild,
-    ViewContainerRef,
-} from '@angular/core';
+    ViewContainerRef
+    } from '@angular/core';
 
-import {WindowRef} from './window.reference';
+import {WindowRef} from '../../shared/window.reference';
 
 export interface ImageSource {
     media: string;
     url: string;
 }
 
-export enum StretchStrategy {
-    crop = 'crop',
-    stretch = 'stretch',
-    original = 'original',
-}
+export type StretchStrategy = 'crop' | 'stretch' | 'original';
```

Comment:
Any specific reason to move away from the enum?

Reply:
Yes. CLI doesn't support the new TypeScript.